### PR TITLE
Example Output for PR379

### PR DESCRIPTION
**Pull request by raptorsun**
Example output of [PR#379](https://github.com/openshift/lightspeed-operator/pull/379)

### CODE REVIEW

File: lightspeed-catalog-4.15/index.yaml [hunk 1]
```diff
@@ -63,3 +114,12 @@ name: preview
...
     skipRange: ">=0.1.0 <0.1.4"
+---
+schema: olm.channel
+package: lightspeed-operator
+name: alpha
+entries:
+  - name: lightspeed-operator.v0.1.3
+    skipRange: ">=0.1.0 <0.1.3"
+  - name: lightspeed-operator.v0.1.4
+    replaces: lightspeed-operator.v0.1.3
```
Comment (xrajesh) [inline lightspeed-catalog-4.15/index.yaml:125]: What are we gaining in this definition  than just specifying `skipRange: ">=0.1.0 <0.1.4" @raptorsun

Comment (raptorsun) [reply lightspeed-catalog-4.15/index.yaml:125]: if we pass only on snapshot, it will list a single item and skip all older versions

**Comment by xrajesh** [comment]
@raptorsun This is just a example output right ? we dont have to merge this .

**Comment by raptorsun** [comment]
> @raptorsun This is just a example output right ? we dont have to merge this .

yes, this is example, we do not neet to merge this PR.

**Comment by raptorsun** [comment]
/hold
just an example

**Comment by raptorsun** [comment]
/close
no longer needed

**Pull request closed without merging**